### PR TITLE
Work on PR demo server deployment

### PR DESCRIPTION
**Pull request by jometzner**
## PR-Type
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no API changes)
[ ] Build-related changes
[X] CI-related changes
[ ] Documentation content changes
[ ] Application / infrastructure changes
[ ] Other: 
## What Is the Current Behavior?

Every synchronized (means no git conflicts exist) GitHub pull request creates demo server installations in Azure to showcase whatever changes. Two installations per PR. One for B2B and one for B2C.

## What Is the New Behavior?

Since PWA 1.0 we can host multiple themes in one installation. Leaving the necessity to deploy two containers. We just need to adjust the link to both.

[AB#69177](https://dev.azure.com/intershop-com/cefd1005-00a7-4c79-927f-a16947d1b2e6/_workitems/edit/69177)

### CODE REVIEW

File: .github/workflows/demo-server.yml [hunk 1]
```diff
@@ -42,51 +48,24 @@ jobs:
...
-        env:
-          APP: '${{ secrets.AZURE_DEMO_RESOURCEGROUP }}-${{ github.event.pull_request.number }}'
         uses: peter-evans/commit-comment@v1
         with:
           body: |
             Azure Demo Servers are available:
-            - [Universal B2C](http://${{ env.APP }}-${{ env.B2C_APP_SUFFIX }}.azurewebsites.net)
-            - [Universal B2B](http://${{ env.APP }}-${{ env.B2B_APP_SUFFIX }}.azurewebsites.net)
+            - [Universal B2C](http://${{ env.AZURE_WEBAPP_NAME }}.azurewebsites.net/home)
+            - [Universal B2B](http://${{ env.AZURE_WEBAPP_NAME }}.azurewebsites.net/home;theme=blue)
```
Comment (dhhyi) [inline .github/workflows/demo-server.yml:71]: I totally forgot that this works now 😅

Comment (jometzner) [reply .github/workflows/demo-server.yml:71]: Means potentially twice as much demo deployments under one service plan

Comment (dhhyi) [reply .github/workflows/demo-server.yml:71]: The service plan is the bill not how many services are deployed within 😉

**Review by dhhyi — APPROVED** [review]
No more tacton?

**Comment by jometzner** [comment]
> No more tacton?

If somebody want's to see this running, he or she needs to shout out. I'll be happy to reactivate this in a demo WebApp.

**Pull request merged**